### PR TITLE
Resolves an issue with RENEWAL_ASPD

### LINKED ISSUE #6900: commenting out renewal_aspd in src/config/renewal.hpp locks all characters to their max aspd - even level 1 novice
* **rAthena Hash**: e56977b5f

* **Client Date**:  2021-11-17

* **Server Mode**:  Renewal

* **Description of Issue**: 
  * Result: commenting out renewal_aspd in src/config/renewal.hpp locks all character to their max aspd - even level 1 novice
  * Expected Result: it should not lock characters to max aspd
  * How to Reproduce: clone latest rAthena -> go to src/config/renewal.hpp -> comment out RENEWAL_ASPD -> compile then run
  * Official Information: no official information, but it is an obvious bug because even level 1 novices have 190 aspd. I tried comparing with having RENEWAL_ASPD enabled and disabled and the bug only shows when it is disabled/commented out

* **Modifications that may affect results**:  no modifications needed, a clean rAthena with commented out RENEWAL_ASPD will do

unknown

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: Fixes #6900

* **Server Mode**: Renewal

* **Description of Pull Request**: 
  * Resolves an issue where if RENEWAL was enabled but RENEWAL_ASPD was disabled, players would receive incorrect ASPD values.
  * Split the ASPD values to their own files so these can be overridden based on the macro defines.
Thanks to @jamesonfajardo!

### CODE REVIEW

**Comment by bgamez23** [comment]
question regarding on this. if some of the jobs use same aspd why don't merge them just like this.
image
Knight and Lord Knight uses same BaseASPD and Rune Knight and Dragon Knight uses same. it might reduce redundant Jobs BaseASPD

**Comment by aleos89** [comment]
I just copied the job_stats file and did a quick macro to delete extra content. I didn't compare to see which values were similar, but yes they can be compressed.

**Comment by slammer86** [comment]
i got idea, why not like this?
if all same except BaseASPD:

just only input :

src/common/database.cpp

+// RENEWAL mode with RENEWAL_ASPD off, load pre-re ASPD
+#if defined(RENEWAL) && !defined(RENEWAL_ASPD)
+				if (importFile.find("job_stats") != std::string::npos)
+					importFile = "db/pre-re/job_stats.yml";
+#endif

**Comment by bgamez23** [comment]
> i got idea, why not like this? if all same except BaseASPD:
> 
> just only input :
> 
> src/common/database.cpp
> 
> // RENEWAL mode with RENEWAL_ASPD off, load pre-re ASPD #if defined(RENEWAL) && !defined(RENEWAL_ASPD) if (importFile.find("job_stats") != std::string::npos) importFile = "db/pre-re/job_stats.yml"; #endif

it will give you conflict for BonusStats

**Comment by bgamez23** [comment]
> 

they are not same. pre-re doesn't support 3rd jobs, latest expanded jobs and 4th jobs. HpFactor, HpIncrease on some job

**Comment by bgamez23** [comment]
missing BaseASPD for Gunslinger and Ninja for pre-re for @aleos89
[Compressed the ASPD class defines](https://github.com/rathena/rathena/pull/6959/commits/2ee6da3082b801334602a42fa0ad0d9e66583cc7)

**Comment by bgamez23** [comment]
omg i forgot to mention the novice high sorry.

**Comment by bgamez23** [comment]
up for this

**Comment by gbasso666** [comment]
seems bugged, my lvl 1 novice has 12 aspd.

File: src/common/database.cpp [hunk 1]
```diff
@@ -190,6 +190,12 @@ void YamlDatabase::parseImports( const ryml::Tree& rootNode ){
 
 #ifdef RENEWAL
 					std::string compiledMode = "Renewal";
+
+					// RENEWAL mode with RENEWAL_ASPD off, load pre-re ASPD
+#ifndef RENEWAL_ASPD
+					if (importFile.find("job_aspd.yml") != std::string::npos)
+						compiledMode = "Prerenewal";
```
Comment (vstumpf) [inline src/common/database.cpp:197]: honestly, this is kind of hacky, but i like it

**Review by vstumpf — APPROVED** [review]

**Pull request merged**